classes/classes/Scenes/Combat/Combat.as

```diff
@@ -3151,16 +3159,15 @@ public class Combat extends BaseContent {
             if (player.hasStatusEffect(StatusEffects.ResonanceVolley)) outputText("Your bow nudges as you ready the next shot, helping you keep your aimed at [monster name].\n\n");
             multiArrowsStrike();
         }
-        if (player.weaponRangePerk == "Throwing"){
+        else if (player.weaponRangePerk == "Throwing"){

```

Comment:
i think you can use isThrownTypeWeapon here

Reply:
This has been changed to use the method